vocode/streaming/synthesizer/azure_synthesizer.py

```diff
@@ -467,6 +474,10 @@ async def dtmf_audio_generator(tone: str):
                     audio_data = await response.read()
                     yield SynthesisResult.ChunkResult(audio_data, True)
 
+        full_ssml = await self.create_ssml(
```

Comment:
this isn't perfectly equivalent to the actual SSML, it handles DTMF tones differently and is likely to drift more in the future

recommend refactoring output the SSML it's creating (probably either update a variable, or yield partial SSML that you concat later) so we have 1 source of truth

Reply:
@bluegoo192 another option (get arthurs approval here idk if its good) could be to just put the dtmf tones however you want, knowing that they are x seconds long (just check).

Reply:
How are dmtf tones used? Are they ever used in conjunction with speech? Why aren't they handled like filler audio?

Reply:
Can we just copy over vocode implementation? 
https://github.com/vocodedev/vocode-core/blob/main/vocode/streaming/action/dtmf.py